styleguide/source/_patterns/01-molecules/08-lists/13-list-topic-related-content/list-topic-related-content.twig

```diff
@@ -0,0 +1,5 @@
+<ul class="list_topic-related-content">
+  <li class="list_item-topic-related-content">{% include 'atoms-h3' with { 'content': listTopicRelatedContent.first.title, 'class': 'list_topic-related-content_title' } %}{% include 'atoms-link-blue' with { 'content': 'Sed nuc', 'class': 'list_topic-related-content_link' } %}</li>
```

Comment:
Please adjust font size - article titles should be 24px size font and are showing up as 30px currently. Also, UX would like a `<h2>` for all article titles where possible, and an `<h3>` is included here